pkg/network/usm/ebpf_ssl.go

```diff
@@ -321,7 +321,24 @@ func (o *sslProgram) Stop() {
 }
 
 func addHooks(m *errtelemetry.Manager, probes []manager.ProbesSelector) func(pathIdentifier, string, string) error {
-	return func(id pathIdentifier, root string, path string) error {
+	symbolsSet := make(common.StringSet, 0)
```

Comment:
I think it's better to keep inside the returned function as it would be saved locally on the stack and make the live easier for the GC

Reply:
that's kind of caching, the result on that code block won't change when the returned function will be called, thus there is no need to regenerate the result